cli/zq/command.go

```diff
@@ -144,19 +144,14 @@ func (c *Command) Run(args []string) error {
 	if err != nil {
 		return err
 	}
-	d := driver.NewCLI(writer)
-	if !c.quiet {
```

Comment:
`Command.quiet` is unused now. Delete it!

Reply:
The WarningsReader is coming back very soon and will wire back into quiet.